econml/dml/dml.py

```diff
@@ -998,6 +996,42 @@ def _gen_model_final(self):
     def _gen_featurizer(self):
         return _RandomFeatures(dim=self.dim, bw=self.bw, random_state=self.random_state)
 
+    def fit(self, Y, T, X=None, W=None, *, sample_weight=None, groups=None,
```

Comment:
Why is this implementation necessary if it's just calling super().fit with all the same arguments?

Reply:
override to not expose the  `freq_weight` and `sample_var` which are existing from super class